### ISSUE TITLE
migrate source url as optional for user created obits

### ISSUE THREAD

**Issue by sagemlee** [sagemlee/obituary_hub#114]

**Comment by sagemlee** [comment]
@stellakunzang can you explain this one a little more?

**Comment by stellakunzang** [comment]
Yeah, I'll make the name clearer and write a detailed description so that in the future when/if we have a chance to return to this, we will know exactly what it entails! Thanks for pointing out the vagueness :)
